server/build_event_protocol/build_event_handler/build_event_handler.go

```diff
@@ -66,6 +67,12 @@ func (b *BuildEventHandler) OpenChannel(ctx context.Context, iid string) interfa
 		targetTracker:           target_tracker.NewTargetTracker(b.env, buildEventAccumulator),
 		hasReceivedStartedEvent: false,
 		eventsBeforeStarted:     make([]*inpb.InvocationEvent, 0),
+		logWriter: chunkstore.New(
+			b.env.GetBlobstore(),
+			&chunkstore.ChunkstoreOptions{
+				WriteBlockSize:       (1 << 20) * 2,
```

Comment:
nit: can you define both of these numbers as constants in a const () block at the top of this file? ie:

const (
  // Chunks will be flushed to blobstore when they reach this size.
  defaultChunkSize: 1_000_000 

  // Chunks will also be flushed to blobstore after this much time
  // passes with no new data being written.
  defaultChunkTimeout: 15 * time.Second
)

Reply:
Yes; didn't know about underscores in numeric literals; that's neat

Reply:
huh, those didn't work for me; nvm

Reply:
OOOOH they work they just don't get syntax highlighted